engines/director/lingo/lingo.cpp

```diff
@@ -988,6 +986,10 @@ Common::String Datum::asString(bool printonly) const {
 			s = *u.s;
 		} else {
 			s = Common::String::format("\"%s\"", u.s->c_str());
+			for (int i = 0; i < (int)s.size(); i++) {
```

Comment:
please use `for (uint i = 0; i < s.size(); i++)`

Reply:
Fixed